traitsui/tests/editors/test_video_editor.py

```diff
@@ -0,0 +1,68 @@
+# (C) Copyright 2004-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+import unittest
+
+import pkg_resources
```

Comment:
Can we use `importlib_resources` instead of `pkg_resources` here? We are trying to remove uses of `pkg_resources` from the codebases.

Reply:
It doesn't appear to be used anywhere else in TraitsUI. Perhaps this should be its own issue?

Reply:
we already have https://github.com/enthought/traitsui/issues/645. resolving here.